Up2dateService/Up2dateService/SetupManager/ChocoHelper.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.ObjectModel;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Up2dateService.SetupManager
+{
+    public static class ChocoHelper
+    {
+        public static bool IsPackageInstalled(string packageName)
+        {
```

Comment:
Add the check for Choco
If not available - return false

Reply:
Done